trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -147,56 +113,59 @@ const getAmendmentById = async (req, res) => {
 };
 
 const getAmendmentByFacilityId = async (req, res) => {
-  const { facilityId } = req.params;
-  const amendment = await api.getAmendmentByFacilityId(facilityId);
+  const { facilityId, amendmentIdOrStatus, type } = req.params;
+  let amendment;
+  if (amendmentIdOrStatus === CONSTANTS.AMENDMENTS.AMENDMENT_QUERY_STATUSES.IN_PROGRESS) {
```

Comment:
Nest `if` conditions are almost two cognitive level, can we change this to `switch` case please.

Reply:
Done with a switch, on the tfm-get-amendments controller as well